docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -102,6 +102,47 @@ Sub-sections related to the Administration Server, specific clusters, or specifi
 
 The elements `serverStartPolicy`, `serverStartState`, `serverPod` and `serverService` are repeated under `adminServer` and under each entry of `clusters` or `managedServers`.  The values directly under `spec` set the defaults for the entire domain.  The values under a specific entry under `clusters` set the defaults for cluster members of that cluster.  The values under `adminServer` or an entry under `managedServers` set the values for that specific server.  Values from the domain scope and values from the cluster (for cluster members) are merged with or overridden by the setting for the specific server depending on the element.  See the [startup and shutdown]({{< relref "/userguide/managing-domains/domain-lifecycle/startup.md" >}}) documentation for details about `serverStartPolicy` combination.
 
+### JVM memory and Java option environment variables
+
+You can use the following environment variables to specify JVM memory and JVM option arguments to WebLogic Server Managed Server and Node Manager instances:
+
+* `JAVA_OPTIONS` : Java options for starting WebLogic Server.
+* `USER_MEM_ARGS` : JVM memory arguments for starting WebLogic Server.
+* `NODEMGR_JAVA_OPTIONS` : Java options for starting Node Manager instance.
+* `NODEMGR_MEM_ARGS` : JVM memory arguments for starting Node Manager instance.
+
+The following behavior occurs depending on whether or not `NODEMGR_JAVA_OPTIONS` and `NODEMGR_MEM_ARGS` are defined:
+
+* If `NODEMGR_JAVA_OPTIONS` is not defined and `JAVA_OPTIONS` is defined, then the `JAVA_OPTIONS` value will be applied to the Node Manager instance.
+* If `NODEMGR_MEM_ARGS` is not defined, then default memory values (-Xms64m -Xmx100m) will be applied to the Node Manager instance.
```

Comment:
The documentation elsewhere states "The `USER_MEM_ARGS` environment variable defaults to `-Djava.security.egd=file:/dev/./urandom` in all WebLogic Server pods and the WebLogic introspection job. It can be explicitly set to another value in your domain resource YAML file using the `env` attribute under the `serverPod` configuration."  It would be good to include this defaulting information here too.

Reply:
Also, please verify that the urandom doc is still correct for both the NM & the WLS processes.

Reply:
We still include the urandom setting by default.